$OpenBSD: patch-ECtools_zarafa-search_ECIndexImporter_cpp,v 1.1 2012/08/30 18:42:26 ajacoutot Exp $

ECIndexImporter.cpp:400:5: error: 'auto_ptr' was not declared in this scope

--- ECtools/zarafa-search/ECIndexImporter.cpp.orig	Wed Aug  8 17:14:58 2012
+++ ECtools/zarafa-search/ECIndexImporter.cpp	Thu Aug 30 17:16:04 2012
@@ -47,6 +47,7 @@
  * 
  */
 
+#include <memory>
 #include "platform.h"
 
 #include "ECIndexImporter.h"
@@ -433,7 +434,7 @@ HRESULT ECIndexImporter::ProcessThread()
     ECIndexDB *lpIndex = NULL;
     ECIndexDB *lpThisIndex = NULL;
     ArchiveItem *lpArchiveItem = NULL;
-    auto_ptr<ArchiveItem> lpStubTarget;
+    std::auto_ptr<ArchiveItem> lpStubTarget;
     
     m_lpLogger->Log(EC_LOGLEVEL_DEBUG, "Importer thread started");
     
